src/app/content/components/Page/highlightManager.ts

```diff
@@ -148,6 +148,7 @@ export default (container: HTMLElement, getProp: () => HighlightProp) => {
       }
       return null;
     },
+    highlighter,
```

Comment:
i don't see you using this anywhere, why did you expose it?

Reply:
thanks, those were leftovers from the previous approach